templates/plate-playground-template/package.json

```diff
@@ -66,6 +66,7 @@
     "@udecode/plate-serializer-md": "^29.0.1",
     "@udecode/plate-tabbable": "^29.0.1",
     "@udecode/plate-table": "^29.0.1",
+    "@udecode/plate-toggle": "^30.1.2",
```

Comment:
Could you remove all the `templates` changes? We need to wait for the release first (CI condition)